homeassistant/components/bring/todo.py

```diff
@@ -43,6 +53,22 @@ async def async_setup_entry(
         for bring_list in coordinator.data.values()
     )
 
+    platform = entity_platform.async_get_current_platform()
+
+    platform.async_register_entity_service(
+        SERVICE_PUSH_NOTIFICATION,
+        vol.Schema(
+            {
+                vol.Required(ATTR_LIST): cv.entities_domain(DOMAIN_TODO),
+                vol.Required(ATTR_NOTIFICATION_TYPE): vol.All(
+                    vol.Upper, cv.enum(BringNotificationType)
+                ),
+                vol.Optional(ATTR_ITEM_NAME): cv.string,
+            }
+        ),
+        "async_send_push_notification",
+    )
```

Comment:
registered it as entity service but the method is never called. Can't figure out why